src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "./IRegistryCoordinator.sol";
+import "./IStakeRegistry.sol";
+import "./IBLSPubkeyRegistry.sol";
+import "./IIndexRegistry.sol";
+
+/**
+ * @title Minimal interface for the `IBLSStakeRegistryCoordinator` contract.
+ * @author Layr Labs, Inc.
+ */
+interface IBLSRegistryCoordinatorWithIndices is IRegistryCoordinator {
+    // STRUCTS
+
+    struct QuorumBitmapUpdate {
+        uint32 updateBlockNumber;
+        uint32 nextUpdateBlockNumber;
+        uint192 quorumBitmap;
+    }
+
+    struct OperatorSetParam {
+        uint32 maxOperatorCount;
+        uint8 kickPercentageOfOperatorStake;
+        uint8 kickPercentageOfAverageStake;
+        uint8 kickPercentageOfTotalStake;
+    }
+
+    struct OperatorKickParam {
```

Comment:
a top-level description for this struct seems appropriate to me

Reply:
and ditto for `OperatorSetParam`

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/598d8c6835e34e1881b409281bc0279bf1e9eb7e